sqlalchemy_ibmi/base.py

```diff
@@ -727,18 +727,14 @@ def create_connect_args(self, url):
         opts.update(url.query)
         if 'port' in opts and opts['port'] is None:
             opts.pop('port')
-        allowed_opts = {'host', 'user', 'password',
+        if 'host' in opts:
+            opts['system'] = opts.pop('host')
```

Comment:
I figured this out too. Not sure why when the docs say it should do this automatically. We can do that above in translate_connect_args, though by adding `host='system'`.